opensearch-observability/src/main/resources/observability-settings.yml

```diff
@@ -7,5 +7,5 @@
 # Settings file for the observability index
 index:
   number_of_shards: "1"
-  auto_expand_replicas: "0-1"
+  auto_expand_replicas: "0-2"
```

Comment:
Some questions out of curiosity:
- what if cluster doesn't have space for 2?
- why do we need to increase this in the first place?

Reply:
1. it will auto expand to maximum 2 replicas depending on cluster size
2. 2 replicas provides more availability, this will be consistent with other plugins (they should be changing soon)